week-5/elephant-talk/fill_data_csv.rb

```diff
@@ -0,0 +1,67 @@
+require "csv"
+
+class FillData
+  $file_products_customers = "products_customers.sql"
+  $file_transactions = "transactions.sql"
+  def products_customers
+    products_name = []
+    products_days = []
+    products_price = []
+    
+    customer_name = []
+    customer_city = []
+    customer_state = []
+    customer_country = []
+    file =  File.open($file_products_customers, "w")
+    CSV.foreach("sales.csv", headers: true) do |row|
+      products_name << row[1]
+      products_days << row[2]
+      products_price << row[3]
+      customer_name << row[7].gsub(/\'/,"''")
```

Comment:
According to my comment before, here you could have done the line 17,18,19 in one line